### DISCUSSION TITLE
Help with credit card expiry input

### DISCUSSION THREAD

**Discussion by subpx** [uNmAnNeR/imaskjs#565] [Q&A]
`new IMask(input, {
      mask: 'MM / YY',
      blocks: {
        YY: {
          mask: IMask.MaskedRange,
          from: 0,
          to: 99
        },
        MM: {
          mask: IMask.MaskedRange,
          from: 1,
          to: 12
        }
      }
    });`

Example: user enters '14' which is a valid MM. the input just displays '14', but what I want it to display is '14 / ' ;
Any help would be much appreciated.

**Comment by subpx** [comment]
Trying to explain better what I want to achieve, hoping someone can offer something.
I want a credit card expiry field with the same behaviour as Stripe.

[https://stripe.dev/elements-examples/](https://stripe.dev/elements-examples/)

When trying to recreate using iMask after entering 2 numbers nothing happens. The Stripe input inserts all the chars " / " until the next 2 numbers.

**Comment by dhaneshmarch** [comment]
@subpx did you get any solution around this, facing the similar issue

**Reply by subpx** [reply]
Yes, you need to use eager mode. Not really documented and possibly wasn’t a feature at the time of submitting issue. I will close this now.
